app/controllers/collections_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CollectionsController < ApplicationController
+  around_action :skip_bullet, if: -> { defined?(Bullet) }
+
+  def show
+    @collection = User.find_by(username: params[:username]).collections.find_by(id: params[:id])
+  end
+
+  private
+
+  def skip_bullet
+    previous_value = Bullet.enable?
+    Bullet.enable = false
+    yield
+  ensure
+    Bullet.enable = previous_value
+  end
```

Comment:
Excellent use of Bullet documentation to allow the N+1 query to be made on line 5. 
I don't feel the query will have a large time-complexity impact since we're returning a relatively small collection of articles. If we were returning hundreds or thousands of records I'd try to optimize more.